tests/code/uid.cpp

```diff
@@ -1,6 +1,8 @@
-#define BOOST_TEST_MODULE UID
-#include <boost/test/unit_test.hpp>
+//#define BOOST_TEST_MODULE UID
+//#include <boost/test/unit_test.hpp>
 
+#define BOOST_TEST_MODULE First_TestSuite
+#include <boost/test/included/unit_test.hpp>
```

Comment:
I'm not sure I understand the reason for this change: did using the `UID` test module name break something?